src/util-action.js

```diff
@@ -0,0 +1,42 @@
+export function createDefaultBailout(storeNode, force) {
+    return (store) => {
+        if (store[storeNode]) {
```

Comment:
I think that the reducer should always be defined and if it is not there is a mapping error and we should give a warning or maybe throw an exception.